### PR TITLE
[FIX] v13 urls

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Was still pointing to old links

**Description of the issue/feature this PR addresses:** This PR updates the urls to v13 and such.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mreficent** [comment]
Also updated [documentation-use](https://github.com/odoo/documentation-user)r in https://github.com/odoo/documentation-user/pull/413.

**Comment by mreficent** [comment]
ping: @mart-e

File: doc/setup/install.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 12.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu 18.04`_ or above.
+Odoo 13.0 'deb' package currently supports `Debian Stretch`_, `Ubuntu 18.04`_ or above.
```
Comment (Elkasitu) [inline doc/setup/install.rst]: To check with @d-fence but I think we support Debian Buster now

File: doc/setup/update.rst [hunk 2]
```diff
@@ -110,7 +110,7 @@ to replace the installation directory with a newer version. First download the l
 from Odoo.com. They are updated daily and include the latest security fixes (see step #1)
 After downloading the package, extract it to a temporary location on your server.
 
-You will get a folder labelled with the version of the source code, for example "odoo-12.0+e.20180719",
+You will get a folder labelled with the version of the source code, for example "odoo-13.0+e.20180719",
```
Comment (mart-e) [inline doc/setup/update.rst]: maybe change the date too

File: doc/setup/install.rst (hunk 1 above)
Comment (d-fence) [reply doc/setup/install.rst]: I confirm. It's Debian Buster now, as it's the new Debian stable

File: doc/setup/update.rst (hunk 2 above)
Comment (mreficent) [reply doc/setup/update.rst]: haha updated

File: doc/setup/install.rst (hunk 1 above)
Comment (mreficent) [reply doc/setup/install.rst]: Change to Buster done!

File: doc/setup/install.rst [hunk 3]
```diff
@@ -200,9 +200,9 @@ If you need this feature, you can install the python module like this:
 Fedora
 ''''''
 
-Odoo 12.0 'rpm' package supports Fedora 26.
+Odoo 13.0 'rpm' package supports Fedora 26.
 As of 2017, CentOS does not have the minimum Python requirements (3.5) for
```
Comment (d-fence) [inline doc/setup/install.rst]: I still need to check the CentOS Stuff. I think that it should work now.

**Comment by Elkasitu** [comment]
robodoo r+

**Comment by Feyensv** [comment]
@robodoo r+

**Comment by Feyensv** [comment]
@robodoo retry

**Pull request closed without merging**